core/services/fluxmonitor/flux_monitor.go

```diff
@@ -654,15 +654,15 @@ func (p *PollingDeviationChecker) SetOracleAddress() error {
 	accounts := p.store.KeyStore.Accounts()
 	for _, acct := range accounts {
 		for _, oracleAddr := range oracleAddrs {
-			if acct.Address == oracleAddr {
+			if acct.Address.Hex() == oracleAddr.Hex() {
```

Comment:
Was this broken??

Reply:
.Equal?

Reply:
no just precarious - the type doesn't have an equal